LIMS_IMAGE/web/laboratoryOrders/models.py

```diff
@@ -149,6 +149,9 @@ def __str__(self):
     status = models.CharField(max_length=50)
     result = models.CharField(max_length=200)
     test_id = models.ForeignKey(TestSample, on_delete=models.CASCADE)
+    pass_fail = models.BooleanField()
+    date_entry = models.DateTimeField(
```

Comment:
I am getting this error when running migrations on startup: 
![image](https://user-images.githubusercontent.com/39775792/154768925-ba329768-ccc5-48c8-96cb-75ec3ec9954f.png)

Reply:
I used `docker-compose up --build` for this to rebuild the db